vendor/engines/saas/app/jobs/saas/trials_create_job.rb

```diff
@@ -0,0 +1,10 @@
+module Saas
+  class TrialsCreateJob < ActiveJob::Base
+
+    def perform(params)
+      user = params['current_user']['login']
+      params['url'] = "/settings/#{user}/trialed"
+      Analytics::PageJob.perform_later(params)
+    end
```

Comment:
I didn't phrase that well - if you try to access an org repo rather than a user repo, does it track `/settings/{org}/trialed` or `/settings/{user}/trialed`? `current_user.login` suggests the latter.